src/commands/junit/api.ts

```diff
@@ -33,13 +33,13 @@ export const uploadJUnitXML = (request: (args: AxiosRequestConfig) => AxiosPromi
 
   const reportTagsAndMetrics: Record<string, any> = {
     tags: jUnitXML.reportTags,
-    metrics: jUnitXML.reportMetrics,
+    metrics: jUnitXML.reportMeasures,
```

Comment:
could we add a quick comment on top of `reportTagsAndMetrics` saying something like:

```javascript
// We can't change `metrics` to `measures` because the backend only accepts `metrics`
```

This is so that we can make sense of the name in the future

Reply:
Good idea!